main.pm

```diff
@@ -424,8 +425,16 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
+        if (!is_staging) {
+            # TEST DEVELOPERS - Put tests that you don't want to run in stagings below here
+            if (!get_var("OFW")) {
+                loadtest "console/aplay.pm";
+            }
+            loadtest "console/yast2_cmdline.pm";
+            loadtest "console/yast2_dns_server.pm";
+            loadtest "console/yast2_nfs_client.pm";
+        }
         if (!get_var("OFW")) {
-            loadtest "console/aplay.pm";
```

Comment:
Have we ever stopped a staging because of an aplay failure, or do we just always go 'oh damn, aplay has failed again' and re-run the test?

I can revert if you want, but in my opinion the aplay test is so untrustworthy, it doesn't make sense to run in stagings which requires a 100% passrate to progress

https://en.wikipedia.org/wiki/False_positive_paradox